spec/requests/users_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe ':index' do
+    before(:each) { get root_path }
+
+    it('status should be :ok') do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it('rendered :index template') do
+      expect(response).to render_template(:index)
+    end
```

Comment:
- [ ] For this action please include a test for `If the response body includes correct placeholder text`.